README_REVIEW.md

```diff
@@ -0,0 +1,92 @@
+# Teal.Garden Track :seedling: :sunny: - Join the Future of Organizations
+
+##### Sustainable Development Goals:
+
+:chart_with_upwards_trend:`SDG 8: Decent Work & Economic Growth`
+:milky_way: `SDG 9: Industry, Innovation and Infrastructure`
+:house_with_garden: `SDG 11: Sustainable Cities and Communities`
+:handshake: `SDG 17: Partnerships for the Goals`
+
+The current health and economic crisis have created vast & sudden change. Now it is up to us to rebuild communities and organizations. Let's take the chance and create the future we want.
+
+Instead of hierarchies that dictate our actions, or economic machineries that force financial rationale over social and ecological wellbeing, we can create work environments where we can bring ourselves as a whole, and collaborate with peers to realize a purpose that we care for deeply.
+
+:thought_balloon: - **"The greatest danger in times of turbulence is not the turbulence, It is to act with yesterday's logic" -- Peter Drucker**
+
+## :question: Questions:
+
+- Which processes and tools have enabled people in your community or workspace to show up as a whole?
+
+- Which practices and processes can you imagine that will bring out the best of your organization in the future?
+
+## :heavy_check_mark: Scope:
+
+:point_right: **First Step:**
+
+- Get comfortable with teal practices & holacracy, take some time and see our resources & explainers:
+
+  - [Basic Explainers](https://hackmd.io/R0EIa_W1QZ2xVa0tSJqk6Q?view)
+
+:point_right: **Second Step:**
+
+- We have created a .json schema file to capture organizational practices and tools that are aligned with the SDG goals.
+
+  - [processSchema.json](https://github.com/deora-earth/tealgarden/blob/develop/processSchema.json)
```

Comment:
maybe we should give the guys a template as google doc, with a lot of questions to help them fill the different sections? like this one: https://docs.google.com/document/d/1kzqyBvQ0_scdZfq2TDWjlCR2wC2dGmB3b1q4JnxS1sk/edit#